### DISCUSSION TITLE
CSV doesn't support Arabic Charachters

### DISCUSSION THREAD

**Discussion by iiDon** [directus/directus#18000] [Archive]
Hi,

Im using directus and i wanted to import csv file with arabic charachters and i got this resault 

![image](https://user-images.githubusercontent.com/65644992/228833619-7e4ce4ab-2322-473d-8944-7a4626c9d018.png)

as you know csv does not support arabic charachtars as well, but xlsx does.

do you have alternetaive solution to import excel file with accepting arabic language?

I want a solution the normal user could use it easliy, i'm a developrt and i can use json. but normal users could not.

Thanks

**Comment by rijkvanzanten** [comment]
What database are you using? This feels like a collation setting problem to me

**Reply by iiDon** [reply]
I don't think the issue from DB, because when i add a recored manually from the system, Arabic Language works fine, but when i export it as csv it carsh.

So both Export And Import not working with csv

Im using Mysql 8 and The collation is the same directus suggested on the documentations.
